concepts/05 UI Components/DataGrid/20 Editing/45 Disable Editing.md

```diff
@@ -0,0 +1,46 @@
+DataGrid has API to disable editing a specific column, row and cell.
+
+### Column
+Set a column's [allowEditing](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#allowEditing) property to **false** to prevent editing its data.
+
+### Row
+Handle the [DataGrid.onEditingStart](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onEditingStart) event, check the **data** parameter's values and assign **true** to **e.cancel** if you want to forbid editing in a specific row.
+
+    <!--JavaScript-->
```

Comment:
Please redo all code snippets for 4 frameworks

Reply:
Thanks. I have took it into account.